ci/task/compile_models.sh

```diff
@@ -0,0 +1,20 @@
+#!/bin/bash
+# Usage:
+# ./compile_models.sh "${target}" ${hf_token}
+set -eo pipefail
+set -x
+
+TARGET=$1
+HF_TOKEN=$2
```

Comment:
Let's follow the convention throughout our testing infra, where we have a predefined [`GPU` flag](https://github.com/mlc-ai/mlc-llm/blob/89867a0be2a066021ea2ed08d3c6b72934891bf7/ci/task/build_lib.sh#L4C1-L6)